TechAndCheck/zenodotus
# Add a media upload loading indicator

This PR creates an animated cog that spins while users upload media to the image search endpoint. 

It also fixes a bug on `master` wherein a user could run media search without uploading any files. 

Closes issue #457

## Testing instructions
1. `rails assets:precompile`
2. Start the Zenodotus server and try searching for media using search bar's explicit button, as well as using drag-and-drop. You should see a loading indicator appear while the media is uploaded.
